src/core/linear_solver/src/preconditioner/4C_linear_solver_preconditioner_muelu.cpp

```diff
@@ -106,7 +106,8 @@ void Core::LinearSolver::MueLuPreconditioner::setup(bool create, Epetra_Operator
 
       for (int block = 0; block < A->rows(); block++)
       {
-        EpetraCrsMatrix crsA(Teuchos::rcpFromRef(*A->matrix(block, block).epetra_matrix()));
+        Teuchos::RCP<Xpetra::CrsMatrix<SC, LO, GO, NO>> xCrsA = Teuchos::make_rcp<EpetraCrsMatrix>(
+            Teuchos::rcp(A->matrix(block, block).epetra_matrix()));
```

Comment:
Why does `xCrsA` need to be an `RCP`?

Reply:
Can the `RCP` be removed here? Not super necessary though ... I just wonder why this change is necessary.